Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -191,13 +159,13 @@ public static MixedRealityManager Instance
                 switch (objects.Length)
                 {
                     case 0:
-                        instance = new GameObject("MixedRealityManager").AddComponent<MixedRealityManager>();
+                        instance = new GameObject(nameof(MixedRealityManager)).AddComponent<MixedRealityManager>();
```

Comment:
Not entirely sure why we need to have this, but not opposed to it.  The MR Manager is the one component in the toolkit that can't really be stripped out,. Unless we are now also looking at a consumable component model?

Reply:
in case we ever rename the class